realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -151,10 +162,9 @@ public boolean isGlobalRealm() {
     private RealmResults<PermissionOffer> offers;
 
     /**
-     * FIXME Javadoc
      * Creates a PermissionManager for the given user.
      *
-     * Implementation notes: This class is thread safe since all public methods have thread confined checks in them
+     * This class is thread safe since all public methods have thread confined checks in them
```

Comment:
My bad, yes we should use thread confined over thread safe IMO, because I agree that thread safe normally means either immutable or synchronized somehow.